examples/models/02-billboard.lisp

```diff
@@ -0,0 +1,38 @@
+(in-package #:cl-user)
+(defpackage claylib/examples/models-2
+  (:use :cl :claylib)
+  (:export :main))
+(in-package #:claylib/examples/models-2)
+
+(defun pathname-from-relative-path (path)
+  "Return a pathname from a path relative to the claylib project root."
+  (asdf:system-relative-pathname :claylib path))
```

Comment:
Is there any reason not to wrap this up as part of claylib, or is it just not thoroughly tested?

Reply:
Well it's not all that useful to a claylib user since all it does is interpret the given `path` as relative to *claylib's* project root - not whatever project they are making. It's useful for any example where we load assets though.

Reply:
Good point. Let's add it to a helpers.lisp file or something then in the claylib/examples package so at least we can reuse it there. For user clarity I think we'd want to explicitly call `(claylib/examples:pathname-from-relative-path ...)` every time just so they know it's not something they should use in their own code.

Reply:
Oh, and maybe shorten the name :grin:

Reply:
Okay I added a separate helpers file for the examples. https://github.com/defun-games/claylib/pull/24/commits/000be404a542a9d110fe9899e34d0a64840ba326